pkg/model/api/v1/common/jsonref.go

```diff
@@ -29,7 +29,7 @@ var (
 
 type JSONRef struct {
 	// Ref is the JSON reference.
-	// That's the only thing used during the marshalling / unmarshalling process.
+	// That's the only thing used during the marshaling / unmarshalling process.
```

Comment:
learned sth today:

> Marshaling is predominantly used in 🇺🇸 American (US) English ( en-US ) while marshalling is predominantly used in 🇬🇧 British English (used in UK/AU/NZ) ( en-GB )

Then I guess you should either change nothing here OR change unmarsha**ll**ing too ;D